pkg/mcs/resource_manager/client/client.go

```diff
@@ -516,21 +521,31 @@ func (gc *groupCostController) handleTokenBucketTrickEvent(ctx context.Context)
 }
 
 func (gc *groupCostController) updateAvgRaWResourcePerSec() {
+	isBurstable := true
 	for typ, counter := range gc.run.resourceTokens {
 		if !gc.calcAvg(counter, getRawResourceValueFromConsumption(gc.run.consumption, typ)) {
 			continue
 		}
+		if counter.limiter.GetBurst() >= 0 {
+			isBurstable = false
+		}
 		log.Debug("[resource group controller] update avg raw resource per sec", zap.String("name", gc.Name), zap.String("type", rmpb.RawResourceType_name[int32(typ)]), zap.Float64("avgRUPerSec", counter.avgRUPerSec))
 	}
+	gc.burstable.Store(isBurstable)
 }
 
 func (gc *groupCostController) updateAvgRUPerSec() {
+	isBurstable := true
 	for typ, counter := range gc.run.requestUnitTokens {
+		if counter.limiter.GetBurst() >= 0 {
```

Comment:
Should we place this check before or after the below check? I noticed that its place is different with `updateAvgRaWResourcePerSec`.

Reply:
Good Catch, it's need before the below check.